.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+    - 12
     - 11
     - 10
     - 9
```

Comment:
We'd better use LTS releases only, so it should be 

```yml
node_js:
    - 12
    - 10
```



https://nodejs.org/ja/about/releases/